frontend/src/Dashboard/CrossingDetailPage/CrossingStatusHistory.js

```diff
@@ -2,8 +2,15 @@ import React, { Component } from 'react';
 
 class CrossingStatusHistory extends Component {
   render() {
+    const { history } = this.props;
+
     return (
-      <div>Crossing Status History: {this.props.crossingId}</div>
+      <div>
+        <div>Crossing Status History: {this.props.crossingId}</div>
```

Comment:
minor, I don't think crossingId is defined as a prop any longer.

Reply:
good catch, I think this one is also fixed in the edit crossing PR